scripts/currentreconcile.py

```
@@ -0,0 +1,435 @@
+#!/usr/bin/env python3
+
+import argparse
+import json
+import os
+import shutil
+import subprocess
+import sys
+from decimal import Decimal, InvalidOperation
+from typing import Dict, List, Optional, Set, Tuple
```

Ruff (0.15.12)

[warning] `typing.Dict` is deprecated, use `dict` instead

(UP035)